doc/refman/RefMan-pro.tex

```diff
@@ -298,15 +298,19 @@ \section{Navigation in the proof tree}
 \begin{Variant}
 \item {\tt Focus {\num}.}\\ 
 This focuses the attention on the $\num^{th}$ subgoal to prove.
-
 \end{Variant}
 
+\emph{This command is deprecated: prefer the use of bullets or
+  focusing brackets instead, including {\tt {\num}: \{}}.
+
 \subsection[\tt Unfocus.]{\tt Unfocus.\comindex{Unfocus}}
 This command restores to focus the goal that were suspended by the
 last {\tt Focus} command.
 
+\emph{This command is deprecated.}
```

Comment:
Please change to "deprecated since Coq 8.8".

Reply:
Done.